apps/explorer/src/index.css

```diff
@@ -14,7 +14,132 @@
     --steel-dark: theme('colors.steel.dark');
 }
 
-body {
-    @apply antialiased;
-    @apply font-sans text-offblack;
+@layer base {
+    body {
+        @apply antialiased;
+        @apply font-sans text-offblack;
+    }
+}
+
+@layer components {
```

Comment:
as a general note, there _could_ be an easier way to implicitly lay out the grid or just use flexbox, but there are 8 different layouts to account for in total so I find this the most digestible